static/library/lollipop_component.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { css, CSSResult, LitElement, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import _ from "lodash";
+import React from "react";
+import ReactDOM from "react-dom";
+
+import tilesCssString from "!!raw-loader!sass-loader!../css/tiles.scss";
+
+import { BarTile, BarTilePropType } from "../js/components/tiles/bar_tile";
+import { DEFAULT_API_ENDPOINT } from "./constants";
+import { convertArrayAttribute } from "./utils";
+
+/**
+ * Web component for rendering a lollipop chart tile.
+ *
+ * Example usage:
+ *
+ * <!-- Show a lollipop chart of population for states in the US -->
+ * <datacommons-lollipop
+ *      title="Population of US States"
+ *      place="country/USA"
+ *      childPlaceType="State"
+ *      variable="Count_Person"
+ * ></datacommons-lollipop>
+ *
+ * <!-- Show a lollipop chart of population for specific US states -->
+ * <datacommons-lollipop
+ *      title="Population of US States"
+ *      variableDcid="Count_Person"
+ *      comparisonPlaces='["geoId/01", "geoId/02"]'
+ * ></datacommons-lollipop>
+ */
+@customElement("datacommons-lollipop")
```

Comment:
Should we combine this component with `datacommons-bar` ? 

maybe:

```html
<datacommons-bar
  title="Population of US States"
  place="country/USA"
  childPlaceType="State"
  variable="Count_Person"
  lollipop
></datacommons-bar>
```

Reply:
Good idea, done as you proposed above.